### PR TITLE
feat(build)!: pretty much complete overhaul

### PR DESCRIPTION
**Pull request by Thomaash**
BREAKING CHANGE: This is still mostly a drop in replacement for the old 4.21 Vis.js but feature wise there were some breaking changes in the past 7 months.

This brings the reexported versions inline with our other packages.

All the issues that have accumulated over the past 7 months (since the last release) are resolved here.

### CODE REVIEW

**Comment by yotamberk** [comment]
Awesome work! I think it might be worth changing the commit from "feat" to the tag that changes the major version since the great work done here is a breaking change

**Comment by mojoaxel** [comment]
> BREAKING CHANGE: This is no longer a drop in replacement for the old 4.21 Vis.js.

I'm confused! To provide a "drop-in replacement" was the whole point of this repository.
Everybody (who can) should use the new libraries. This was especially for the people/projects that can/will not change there code.

**Comment by Thomaash** [comment]
I get your point and it's a good point. However this broke about 7 months ago (last release) and nobody had the time to fix it. And since we are making more and more breaking changes it will be even more demanding to keep this as a drop in replacement.

**Comment by Thomaash** [comment]
> Awesome work! I think it might be worth changing the commit from "feat" to the tag that changes the major version since the great work done here is a breaking change

Thanks.

Regarding versioning: The prefixes are irrelevant (feat, fix or even perf can be major if appropriate). What causes major version releases is the `BREAKING CHANGE:` at the start of the commit message body.

File: README.md [hunk 1]
```diff
@@ -1,7 +1,5 @@
 # vis-charts
 
-**This is a "plug'n & play replacement" for [vis](https://github.com/almende/vis), wich is no longer maintained**
-
```
Comment (mojoaxel) [inline README.md:4]: @Thomaash Can you please also remove/update the "Usage" section in this README (and maybe even remove the greenkeeper badge). Thx!

**Comment by Thomaash** [comment]
Okay, so I reworked this to a compromise solution. Everything new is simply reexported but the legacy, deprecated and no longer available exports (`vis.network`, `vis.util`, `vis.DOMutil`…) are still available through this. The package format is also unchanged now (except that even CSS has maps but that's not a breaking change).

Regarding the fancy Rollup config: It makes things so much easier and especially changes in config. I change it in one place and the rest is just updates (Renovate handles most of that). Copy pasting the stuff all over the place every time something changes is a nightmare.

**Pull request merged**